### DISCUSSION TITLE
Keep a service running forever using the NextJs backend server

### DISCUSSION THREAD

**Discussion by akshaaatt** [vercel/next.js#40916] [Help]
### Describe the feature you'd like to request

I want to check the AWS SQS intervals for my backend. Where do I initialize it such that it keeps on checking even when the website is closed?

### Describe the solution you'd like

A way to keep this service running forever.

### Describe alternatives you've considered

Creating a custom server from the nextjs app works but that slows down the app.

**Comment by 1finedev** [comment]
check this out
https://vercel.com/docs/v2/platform/limits#streaming-responses
